### ISSUE TITLE
Error on device edit

### ISSUE THREAD

**Issue by Cyberzoid1** [uidaho/smartsettia#102]
Error when trying to edit a device from the manage devices page

## Steps to Reproduce (for bugs)
On master 907f989 merged with #100 
composer updated and migrated
Deleted framework/cache

Able to edit device on /device/57
http://smartsettia-nkrenowicz.c9users.io/device/57/edit

```json
Illuminate\Database\QueryException thrown with message "SQLSTATE[42S22]: Column not found: 1054 Unknown column 'image.updated_at' in 'field list' (SQL: select `devices`.`id`, `name`, `location_id`, `cover_command`, `cover_status`, `open_time`, `close_time`, `update_rate`, `image_rate`, `sensor_rate`, `last_network_update_at`, `image`.`updated_at` as `image_updated_at` from `devices` where `devices`.`id` = 57 and `devices`.`deleted_at` is null limit 1)"

Stacktrace:
#64 Illuminate\Database\QueryException in /home/ubuntu/workspace/vendor/laravel/framework/src/Illuminate/Database/Connection.php:664
#63 PDOException in /home/ubuntu/workspace/vendor/laravel/framework/src/Illuminate/Database/Connection.php:326
#62 PDO:prepare in /home/ubuntu/workspace/vendor/laravel/framework/src/Illuminate/Database/Connection.php:326
#61 Illuminate\Database\Connection:Illuminate\Database\{closure} in /home/ubuntu/workspace/vendor/laravel/framework/src/Illuminate/Database/Connection.php:657
#60 Illuminate\Database\Connection:runQueryCallback in /home/ubuntu/workspace/vendor/laravel/framework/src/Illuminate/Database/Connection.php:624
#59 Illuminate\Database\Connection:run in /home/ubuntu/workspace/vendor/laravel/framework/src/Illuminate/Database/Connection.php:333
#58 Illuminate\Database\Connection:select in /home/ubuntu/workspace/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1705

```

**Comment by brec9824** [comment]
Oh it is because of my select statement on the device controller is using the one created for the dashboard. I will fix.

**Comment by brec9824** [comment]
Closing, fixed with pull request #103
